Website: Add note w.r.t. docker container (#4228)
CodaProtocol/coda: frontend/website2/pages/docs/getting-started.mdx

<Alert>

>   These instructions are tempporarily for a **release-cantidate**. They will be changed on January 24th with the launch of Phase 3.

Reviewer: ```suggestion
  These instructions are temporarily for a **release-candidate**. They will be changed on January 24th with the launch of Phase 3.
```

Next to download, start and connect to the image by running the command below:

```

> docker run -it -d --entrypoint /bin/bash  codaprotocol/coda-daemon:0.0.11-beta1-release-0.0.12-beta-493b4c6

Reviewer: ```suggestion
docker run -d --entrypoint /bin/bash  codaprotocol/coda-daemon:0.0.11-beta1-release-0.0.12-beta-493b4c6
```

Reviewer: ignore this one if you want @figitaki

```

> This starts a new container with the latest release installed and opens a shell. All the following commands should be run inside the Docker container. **NOTE** We are running in detatched mode which means closing the terminal will not stop the docker container.

Reviewer: ```suggestion
This starts a new container with the latest release installed and opens a shell. All the following commands should be run inside the Docker container. **NOTE** We are running in detatched mode which means closing the terminal will not stop the docker container.

If you'd like to run the container interactively, you can add `-it` to the command above. Note that you can always detach the container with the key sequence `ctrl-p` `ctrl-q`.
```
